src/store/storage-controller.ts

```diff
@@ -111,19 +109,46 @@ export class StorageController {
     return messages;
   }
 
-  public static async delete(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async delete(
     tenant: string,
     message: BaseMessage
   ): Promise<void> {
     const messageCid = await Message.getCid(message);
 
     if (message.descriptor.dataCid !== undefined) {
-      await dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
+      await this.dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
     }
 
-    await messageStore.delete(tenant, messageCid);
+    await this.messageStore.delete(tenant, messageCid);
+  }
+
+  public getMessage(tenant: string, messageCid: string): Promise<BaseMessage|undefined> {
```

Comment:
i think we normally put spaces around the ` | `
```suggestion
  public getMessage(tenant: string, messageCid: string): Promise<BaseMessage | undefined> {
```

Reply:
Obsoleted by https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/f872016b3691c4b2f4465ca59aaa33507847a820